staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/types.go

```diff
@@ -149,6 +150,14 @@ const MaxSeatSeconds = SeatSeconds(math.MaxUint64)
 // MinSeatSeconds is the lowest representable value of SeatSeconds
 const MinSeatSeconds = SeatSeconds(0)
 
+// RDecrement is the amount by which the progress meter R is wound backwards
+// when needed to avoid overflow.
+const RDecrement = MaxSeatSeconds / 2
```

Comment:
This is an implementation detail - let's move these constants to queueset.go and make them private.